libs/utils/utils.js

```diff
@@ -608,12 +608,16 @@ export function decorateAutoBlock(a) {
 }
 
 export function decorateLinks(el) {
+  const config = getConfig();
   decorateImageLinks(el);
   const anchors = el.getElementsByTagName('a');
   return [...anchors].reduce((rdx, a) => {
     appendHtmlToLink(a);
     a.href = localizeLink(a.href);
     decorateSVG(a);
+    if (config.env?.name === 'stage' && config.stageDomainsMap?.[a.hostname]) {
```

Comment:
The only part that I'm not happy with, is loading extra code for stage. But I don't think there is a (good) way around it.

Reply:
True, I also don't think there is a good way around it. At least the extra code is minimal.